### PR TITLE
Fix bug in ListMembers widget

### PR DESCRIPTION
**Pull request by emmarogge**
Fix bug that caused ListMembers widget to fail to return correct HTML.
See working [ListGroups widget](https://screenshot.googleplex.com/BfP3VMdkfcjUVdu)
See working [ListMembers widget](https://screenshot.googleplex.com/6L7KvmL4GxNadQf)

### CODE REVIEW

**Review by wnojopra — APPROVED** [review]
LGTM. While you're here, could you also call `.strip()` on the inputs? If one were to copy a group from the dataframe listing your group memberships and then paste it into ListMembersWidget, the whitespace messes it up:
image

File: first_hour_on_vwb/widget_utils.py [hunk 1]
```diff
@@ -44,6 +43,19 @@ def users_table(json_string):
...
 
+def list_group_members(json_string):
+    """ List members of a VWB group in HTML table form.
+    """
+    html = f"""<table style='margin: 0 auto,text-align: left'>"""
+    json_data = json.loads(json_string)
+    html += "<th>EMAIL</th>"
+    html += "<th>POLICIES</th>"
+    for row in json_data:
+        html += "<tr>"
```
Comment (johntbates) [inline first_hour_on_vwb/widget_utils.py:54]: Should there be a corresponding `</tr>` tag?

File: first_hour_on_vwb/working_with_groups.ipynb
Comment (johntbates) [inline first_hour_on_vwb/working_with_groups.ipynb:1]: Not sure why it's not letting me comment inline, but I'd split the `terra_command` list over several lines for readability.

**Review by johntbates — APPROVED** [review]

File: first_hour_on_vwb/widget_utils.py (hunk 1 above)
Comment (emmarogge) [reply first_hour_on_vwb/widget_utils.py:54]: Oddly, it doesn't seem to make a difference whether the `</tr>` tag is added. The output appears the same with and without it for both methods. To be safe, I have now added the closing tags anyway.

File: first_hour_on_vwb/working_with_groups.ipynb
Comment (emmarogge) [reply first_hour_on_vwb/working_with_groups.ipynb:1]: Fixed!

File: first_hour_on_vwb/widget_utils.py (hunk 1 above)
Comment (wnojopra) [reply first_hour_on_vwb/widget_utils.py:54]: Yea there isn't an official HTML "compiler" or anything, so most modern browsers are very "loose" with what they're willing to render.

**Comment by emmarogge** [comment]
> LGTM. While you're here, could you also call `.strip()` on the inputs? 
Excellent idea -- I have now done so!

**Pull request merged**